servatrice/src/serversocketinterface.cpp

```diff
@@ -863,6 +895,9 @@ Response::ResponseCode ServerSocketInterface::cmdRegisterAccount(const Command_R
         return Response::RespUsernameInvalid;
     }
 
+    if (userName == "Servatrice" || userName == "servatrice")
```

Comment:
This should probably be swapped for
`if (userName.toLower() == "servatrice")` as there are multiple ways to write it out...